test/common/common.go

```diff
@@ -122,53 +122,6 @@ func RemoveExtension(shoot *gardencorev1beta1.Shoot, extensionType string) {
 	})
 }
 
-// WaitUntilRegistryCacheConfigurationsAreApplied waits until the configure-containerd-registries.service systemd unit gets inactive on the Nodes.
-// The unit will be in active state until it configures all registry caches.
-func WaitUntilRegistryCacheConfigurationsAreApplied(ctx context.Context, log logr.Logger, shootClient kubernetes.Interface) {
```

Comment:
We don't need anymore this func because now the registry configuration is coupled to the Shoot reconciliation, right? If GNA cannot configure the registries, the Shoot reconciliation won't succeed. Nice 👍